src/material-experimental/theming/_theming.scss

```diff
@@ -0,0 +1,142 @@
+@use 'sass:list';
+@use 'sass:map';
+@use 'sass:meta';
+@use 'sass:string';
+@use '@angular/material' as mat;
+
+// Whether to throw an error when a required dep is not configured. If false, the dep will be
+// automatically configured instead.
+$_error-on-missing-dep: false;
```

Comment:
I would have thought that this would be a per-dependency setting? Or is this just for prototyping purposes?

Reply:
I assumed we'd settle on a way to handle it eventually. I don't know if it would make sense to error for some and not others.